iis/images/IIS-Overview(cloned)-dp-remake--2020-10-26T10_51_02 (3).json

```diff
@@ -0,0 +1 @@
indexes":[],"query":"\"iis\"","sort":{"column":"time","order":"desc"},"columns":["core_host","core_service","log_http.response_time"],"show_date_column":true,"show_message_column":true,"message_display":"expanded-md"}},{"id":1325149753070404,"layout":{"x":171,"y":0,"width":48,"height":6},"definition":{"type":"note","content":"IIS Log Events","background_color":"gray","font_size":"18","text_align":"center","show_tick":false,"tick_pos":"50%","tick_edge":"bottom"}},{"id":4233242180206356,"layout":{"x":0,"y":13,"width":28,"height":41},"definition":{"type":"note","content":"Internet Information Services (IIS) is a flexible, general-purpose web server from Microsoft that runs on Windows systems to serve requested HTML pages or files.\n\nThis dashboard provides a high-level view of your IIS sites' status, plus breakdowns of requests and detailed views into network metrics. The IIS check is packaged with the Agent. To start gathering your IIS metrics and logs, install the Agent on your IIS servers. Further reading on IIS monitoring and integrations:\n\n[Datadog's IIS integration docs](https://docs.datadoghq.com/integrations/iis/?tab=host)\n\n[Key IIS metrics to monitor](https://www.datadoghq.com/blog/iis-metrics/)\n\n[Collecting metrics with IIS monitoring tools](https://www.datadoghq.com/blog/iis-monitoring-tools/)\n\n[IIS monitoring with Datadog](https://www.datadoghq.com/blog/iis-monitoring-datadog/)\n\nClone this template dashboard to make changes and add your own graph widgets. (cloned)\n","background_color":"yellow","font_size":"14","text_align":"left","show_tick":false,"tick_pos":"50%","tick_edge":"left"}},{"id":1097058623080560,"layout":{"x":171,"y":56,"width":48,"height":14},"definition":{"type":"note","content":"Since several key IIS metrics are only available from logs, it’s important to collect and analyze your logs in order to get full visibility into your deployment. You can configure the Datadog Agent to collect logs from IIS, parse them, and send them to Datadog. Once Datadog is ingesting your logs, you can use that data to identify trends and get alerted to IIS performance issues. You can check out our blog regarding [IIS Log Configuration with Datadog](https://www.datadoghq.com/blog/iis-monitoring-datadog/#configure-iis-log-collection), or jump to our documentation on [IIS log collection](https://docs.datadoghq.com/integrations/iis/?tab=host#log-collection).","background_color":"yellow","font_size":"14","text_align":"left","show_tick":false,"tick_pos":"50%","tick_edge":"left"}},{"id":215670117005800,"definition":{"type":"timeseries","requests":[{"q":"sum:iis.net.num_connections{$scope,!site:total}"}],"title":"IIS connections","title_size":"16","title_align":"left","show_legend":false,"legend_size":"0"},"layout":{"x":29,"y":35,"width":28,"height":14}}],"template_variables":[{"name":"scope","default":"*","prefix":null},{"name":"var","default":"*","prefix":null}],"layout_type":"free","is_read_only":true,"notify_list":[],"id":"qrn-bfa-2g2"}
```

Comment:
```suggestion
{"title":"IIS - Overview","description":"This dashboard provides a high-level view of your IIS sites' status, plus breakdowns of requests and detailed views into network metrics. Further reading on IIS monitoring:\n\n- [Datadog's IIS integration docs](https://docs.datadoghq.com/integrations/iis/)\n\n- [Key IIS metrics to monitor](https://www.datadoghq.com/blog/iis-metrics/)\n\n- [Collecting metrics with IIS monitoring tools](https://www.datadoghq.com/blog/iis-monitoring-tools/)\n\n- [IIS monitoring with Datadog](https://www.datadoghq.com/blog/iis-monitoring-datadog/)\n\nClone this template dashboard to make changes and add your own graph widgets. (cloned)\n\ndatadog_demo_keep:true","widgets":[{"id":0,"definition":{"type":"timeseries","requests":[{"q":"avg:iis.users.anon{$scope,!site:total} by {site}","display_type":"line","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"}}],"custom_links":[],"title":"Anonymous","title_size":"16","title_align":"left","time":{"live_span":"4h"},"show_legend":false,"legend_size":"0"},"layout":{"x":134,"y":11,"width":36,"height":15}},{"id":1,"layout":{"x":97,"y":0,"width":73,"height":6},"definition":{"type":"note","content":"Requests per second","background_color":"gray","font_size":"18","text_align":"center","show_tick":false,"tick_pos":"50%","tick_edge":"bottom"}},{"id":2,"definition":{"type":"timeseries","requests":[{"q":"avg:iis.requests.cgi{$scope,!site:total} by {site}","display_type":"line","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"}}],"custom_links":[],"title":"CGI","title_size":"16","title_align":"left","time":{"live_span":"4h"},"show_legend":false,"legend_size":"0"},"layout":{"x":134,"y":46,"width":36,"height":15}},{"id":3,"definition":{"type":"timeseries","requests":[{"q":"avg:iis.requests.isapi{$scope,!site:total} by {site}","display_type":"line","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"}}],"custom_links":[],"title":"ISAPI","title_size":"16","title_align":"left","time":{"live_span":"4h"},"show_legend":false,"legend_size":"0"},"layout":{"x":134,"y":61,"width":36,"height":15}},{"id":4,"definition":{"type":"timeseries","requests":[{"q":"avg:iis.httpd_request_method.get{$scope,!site:total} by {site}","display_type":"line","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"}}],"custom_links":[],"title":"GET","title_size":"16","title_align":"left","time":{"live_span":"4h"},"show_legend":false,"legend_size":"0"},"layout":{"x":97,"y":26,"width":36,"height":15}},{"id":5,"layout":{"x":134,"y":41,"width":36,"height":5},"definition":{"type":"note","content":"By Interface","background_color":"blue","font_size":"16","text_align":"center","show_tick":true,"tick_pos":"50%","tick_edge":"bottom"}},{"id":6,"layout":{"x":97,"y":21,"width":36,"height":5},"definition":{"type":"note","content":"By HTTP method","background_color":"blue","font_size":"16","text_align":"center","show_tick":true,"tick_pos":"50%","tick_edge":"bottom"}},{"id":7,"layout":{"x":134,"y":6,"width":36,"height":5},"definition":{"type":"note","content":"By User Type","background_color":"blue","font_size":"16","text_align":"center","show_tick":true,"tick_pos":"50%","tick_edge":"bottom"}},{"id":8,"definition":{"type":"timeseries","requests":[{"q":"avg:iis.httpd_request_method.put{$scope,!site:total} by {site}","display_type":"line","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"}}],"custom_links":[],"title":"PUT","title_size":"16","title_align":"left","time":{"live_span":"4h"},"show_legend":false,"legend_size":"0"},"layout":{"x":97,"y":41,"width":36,"height":15}},{"id":9,"definition":{"type":"timeseries","requests":[{"q":"avg:iis.httpd_request_method.post{$scope,!site:total} by {site}","display_type":"line","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"}}],"custom_links":[],"title":"POST","title_size":"16","title_align":"left","time":{"live_span":"4h"},"show_legend":false,"legend_size":"0"},"layout":{"x":97,"y":56,"width":36,"height":15}},{"id":10,"definition":{"type":"timeseries","requests":[{"q":"avg:iis.httpd_request_method.delete{$scope,!site:total} by {site}","display_type":"line","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"}}],"custom_links":[],"title":"DELETE","title_size":"16","title_align":"left","time":{"live_span":"4h"},"show_legend":false,"legend_size":"0"},"layout":{"x":97,"y":71,"width":36,"height":15}},{"id":11,"definition":{"type":"timeseries","requests":[{"q":"avg:iis.users.nonanon{$scope,!site:total} by {site}","display_type":"line","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"}}],"custom_links":[],"title":"Non-anonymous","title_size":"16","title_align":"left","time":{"live_span":"4h"},"show_legend":false,"legend_size":"0"},"layout":{"x":134,"y":26,"width":36,"height":15}},{"id":12,"definition":{"type":"timeseries","requests":[{"q":"avg:iis.net.num_connections{$scope,!site:total} by {site}","display_type":"bars","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"}}],"custom_links":[],"title":"Current active connections by site","title_size":"16","title_align":"left","time":{"live_span":"4h"},"show_legend":false,"legend_size":"0"},"layout":{"x":58,"y":21,"width":38,"height":15}},{"id":13,"definition":{"type":"timeseries","requests":[{"q":"avg:iis.net.bytes_sent{$scope,!site:total} by {site}","display_type":"line","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"}}],"custom_links":[],"title":"Bytes sent by site","title_size":"16","title_align":"left","time":{"live_span":"4h"},"show_legend":false,"legend_size":"0"},"layout":{"x":58,"y":36,"width":38,"height":15}},{"id":14,"definition":{"type":"timeseries","requests":[{"q":"avg:iis.net.bytes_rcvd{$scope,!site:total} by {site}","display_type":"line","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"}}],"custom_links":[],"title":"Bytes received by site","title_size":"16","title_align":"left","time":{"live_span":"4h"},"show_legend":false,"legend_size":"0"},"layout":{"x":58,"y":51,"width":38,"height":15}},{"id":15,"layout":{"x":58,"y":6,"width":38,"height":15},"definition":{"title":"Current total number of active connections","title_size":"16","title_align":"left","time":{"live_span":"10m"},"type":"query_value","requests":[{"q":"sum:iis.net.num_connections{$scope}","aggregator":"avg"}],"autoscale":true,"custom_links":[],"precision":2}},{"id":16,"layout":{"x":97,"y":6,"width":36,"height":15},"definition":{"title":"Current requests volume (all methods)","title_size":"16","title_align":"left","time":{"live_span":"10m"},"type":"query_value","requests":[{"q":"sum:iis.httpd_request_method.put{$scope}+sum:iis.httpd_request_method.get{$scope}+sum:iis.httpd_request_method.post{$scope}+sum:iis.httpd_request_method.delete{$scope}","aggregator":"avg"}],"autoscale":true,"custom_links":[],"precision":2}},{"id":17,"layout":{"x":29,"y":0,"width":28,"height":12},"definition":{"title":"Sites up","title_size":"16","title_align":"center","type":"check_status","check":"iis.site_up","grouping":"cluster","group":"$scope","group_by":[],"tags":["$scope"]}},{"id":18,"layout":{"x":58,"y":0,"width":38,"height":6},"definition":{"type":"note","content":"Network traffic","background_color":"gray","font_size":"18","text_align":"center","show_tick":false,"tick_pos":"50%","tick_edge":"bottom"}},{"id":19,"layout":{"x":0,"y":0,"width":28,"height":13},"definition":{"type":"image","url":"/static/images/saas_logos/small/iis@2x.png","sizing":"center"}},{"id":20,"definition":{"type":"toplist","requests":[{"q":"top(avg:iis.uptime{$scope,!site:total} by {site}, 10, 'mean', 'desc')","style":{"palette":"dog_classic"}}],"custom_links":[],"title":"IIS sites by uptime","title_size":"16","title_align":"left","time":{"live_span":"10m"}},"layout":{"x":29,"y":12,"width":28,"height":23}},{"id":21,"definition":{"type":"timeseries","requests":[{"q":"avg:iis.errors.not_found{$scope,!site:total} by {site}","display_type":"line","style":{"palette":"warm","line_type":"solid","line_width":"normal"}}],"custom_links":[],"title":"404 (Not found) responses by site","title_size":"16","title_align":"left","time":{"live_span":"4h"},"show_legend":false,"legend_size":"0"},"layout":{"x":29,"y":49,"width":28,"height":15}},{"id":22,"definition":{"type":"timeseries","requests":[{"q":"avg:iis.net.files_sent{$scope,!site:total} by {site}","display_type":"line","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"}}],"custom_links":[],"title":"Files sent by site","title_size":"16","title_align":"left","time":{"live_span":"4h"},"show_legend":false,"legend_size":"0"},"layout":{"x":58,"y":66,"width":38,"height":15}},{"id":23,"definition":{"type":"timeseries","requests":[{"q":"avg:iis.net.files_rcvd{$scope,!site:total} by {site}","display_type":"line","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"}}],"custom_links":[],"title":"Files received by site","title_size":"16","title_align":"left","time":{"live_span":"4h"},"show_legend":false,"legend_size":"0"},"layout":{"x":58,"y":81,"width":38,"height":15}},{"id":695270390750234,"layout":{"x":171,"y":6,"width":48,"height":50},"definition":{"title":"Log Stream","title_size":"16","title_align":"left","type":"log_stream","indexes":[],"query":"\"iis\"","sort":{"column":"time","order":"desc"},"columns":["core_host","core_service","log_http.response_time"],"show_date_column":true,"show_message_column":true,"message_display":"expanded-md"}},{"id":1325149753070404,"layout":{"x":171,"y":0,"width":48,"height":6},"definition":{"type":"note","content":"IIS Log Events","background_color":"gray","font_size":"18","text_align":"center","show_tick":false,"tick_pos":"50%","tick_edge":"bottom"}},{"id":4233242180206356,"layout":{"x":0,"y":13,"width":28,"height":41},"definition":{"type":"note","content":"Internet Information Services (IIS) is a flexible, general-purpose web server from Microsoft that runs on Windows systems to serve requested HTML pages or files.\n\nThis dashboard provides a high-level view of your IIS sites' status, plus breakdowns of requests and detailed views into network metrics. The IIS check is packaged with the Agent. To start gathering your IIS metrics and logs, install the Agent on your IIS servers. Further reading on IIS monitoring and integrations:\n\n[Datadog's IIS integration docs](https://docs.datadoghq.com/integrations/iis/?tab=host)\n\n[Key IIS metrics to monitor](https://www.datadoghq.com/blog/iis-metrics/)\n\n[Collecting metrics with IIS monitoring tools](https://www.datadoghq.com/blog/iis-monitoring-tools/)\n\n[IIS monitoring with Datadog](https://www.datadoghq.com/blog/iis-monitoring-datadog/)\n\nClone this template dashboard to make changes and add your own graph widgets. (cloned)\n","background_color":"yellow","font_size":"14","text_align":"left","show_tick":false,"tick_pos":"50%","tick_edge":"left"}},{"id":1097058623080560,"layout":{"x":171,"y":56,"width":48,"height":14},"definition":{"type":"note","content":"Since several key IIS metrics are only available from logs, it’s important to collect and analyze your logs in order to get full visibility into your deployment. You can configure the Datadog Agent to collect logs from IIS, parse them, and send them to Datadog. Once Datadog is ingesting your logs, you can use that data to identify trends and get alerted to IIS performance issues. You can check out our blog regarding [IIS Log Configuration with Datadog](https://www.datadoghq.com/blog/iis-monitoring-datadog/#configure-iis-log-collection), or jump to our documentation on [IIS log collection](https://docs.datadoghq.com/integrations/iis/?tab=host#log-collection).","background_color":"yellow","font_size":"14","text_align":"left","show_tick":false,"tick_pos":"50%","tick_edge":"left"}},{"id":215670117005800,"definition":{"type":"timeseries","requests":[{"q":"sum:iis.net.num_connections{$scope,!site:total}"}],"title":"IIS connections","title_size":"16","title_align":"left","show_legend":false,"legend_size":"0"},"layout":{"x":29,"y":35,"width":28,"height":14}}],"template_variables":[{"name":"scope","default":"*","prefix":null},{"name":"var","default":"*","prefix":null}],"layout_type":"free","is_read_only":true,"notify_list":[],"id":"qrn-bfa-2g2"}
```

Let's update the title to remain the same.
It doesn't seem like this dashboard was exported via the `ddev meta dash export` command. Can you use the CLI command so the payload gets automatically formatted and the necessary fields are included. It will automatically include the payload to the right location `iis/assets/dashboards/` and update the [manifest.json field](https://github.com/DataDog/integrations-core/blob/master/iis/manifest.json#L26)? See this:

https://datadoghq.dev/integrations-core/guidelines/dashboards/#exporting-a-dashboard-payload